src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -189,9 +234,16 @@
 	
+	
+	//Scythes
 	public static final RegistryObject<Item> SCYTHE = REGISTER.register("scythe", () -> new WeaponItem(new WeaponItem.Builder(Tiers.IRON, 4, -2.6F).efficiency(1.5F).disableShield().set(MSItemTypes.SICKLE_TOOL), new Item.Properties().tab(MSItemGroup.WEAPONS)));
 	public static final RegistryObject<Item> EIGHTBALL_SCYTHE = REGISTER.register("eightball_scythe", () -> new WeaponItem(new WeaponItem.Builder(Tiers.IRON, 3, -2.6F).efficiency(1.5F).disableShield().set(MSItemTypes.SICKLE_TOOL).add(OnHitEffect.RANDOM_DAMAGE).set(ItemRightClickEffect.EIGHTBALL), new Item.Properties().defaultDurability(600).tab(MSItemGroup.WEAPONS)));
-	
+	public static final RegistryObject<Item> TIME_FLAYER = REGISTER.register("time_flayer", () -> new WeaponItem(new WeaponItem.Builder(Tiers.DIAMOND, 3, -2.6F).efficiency(1.5F).disableShield().set(MSItemTypes.SICKLE_TOOL).add(OnHitEffect.TIME_SLOWNESS_AOE).add(OnHitEffect.RANDOM_DAMAGE), new Item.Properties().tab(MSItemGroup.WEAPONS).rarity(Rarity.RARE)));
+	public static final RegistryObject<Item> SUNRAY_HARVESTER = REGISTER.register("sunray_harvester", () -> new WeaponItem(new WeaponItem.Builder(Tiers.DIAMOND, 4, -2.6F).efficiency(1.5F).disableShield().set(MSItemTypes.SICKLE_TOOL).add(OnHitEffect.setOnFire(35)), new Item.Properties().defaultDurability(1200).tab(MSItemGroup.WEAPONS)));
+	public static final RegistryObject<Item> GREEN_SUN_RAYREAPER = REGISTER.register("green_sun_rayreaper", () -> new WeaponItem(new WeaponItem.Builder(Tiers.NETHERITE, 7, -2.6F).efficiency(1.5F).disableShield().set(MSItemTypes.SICKLE_TOOL).add(OnHitEffect.setOnFire(10)), new Item.Properties().tab(MSItemGroup.WEAPONS).fireResistant().rarity(Rarity.RARE)));
+	public static final RegistryObject<Item> SKAITHE = REGISTER.register("skaithe", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.REGI_TIER, 8, -2.6F).efficiency(1.5F).disableShield().set(MSItemTypes.SICKLE_TOOL), new Item.Properties().tab(MSItemGroup.WEAPONS).rarity(Rarity.UNCOMMON)));
+	public static final RegistryObject<Item> HELLBRINGERS_HOE_INACTIVE = REGISTER.register("hellbringers_hoe_inactive", () -> new WeaponItem(new WeaponItem.Builder(Tiers.NETHERITE, 3, -2.4F).efficiency(10.0F).set(MSItemTypes.SICKLE_TOOL).set(ItemRightClickEffect.switchTo(MSItems.HELLBRINGERS_HOE_ACTIVE)), new Item.Properties().tab(MSItemGroup.WEAPONS)));
+	public static final RegistryObject<Item> HELLBRINGERS_HOE_ACTIVE = REGISTER.register("hellbringers_hoe_active", () -> new WeaponItem(new WeaponItem.Builder(Tiers.NETHERITE, 2, -2.4F).efficiency(10.0F).set(MSItemTypes.SICKLE_TOOL).set(ItemRightClickEffect.switchTo(MSItems.HELLBRINGERS_HOE_INACTIVE)).add(OnHitEffect.setOnFire(10)), new Item.Properties()));
```

Comment:
w0w i c0uldve sw0rn i replied t0 this. wtf.
guess ill type it 0ut again??
the th0ught behind the hellbringers h0e is that all h0es will be scythes, but n0t all scythes will be h0es. like h0w all sh0vels are sp00ns currently (0r the 0ther way ar0und).
s0 it will be treated as a h0e with regards t0 stats, but will be able t0 be used in b0th h0ekind and scythekind specibi :)